cdap-proto/src/main/java/io/cdap/cdap/proto/security/Credential.java

```diff
@@ -84,10 +84,31 @@ public static CredentialType fromQualifiedName(String qualifiedName) {
 
   private final String value;
   private final CredentialType type;
+  private final String expirationTimeSecs;
```

Comment:
Can we use `Instant` instead of `String` type to avoid ambiguity?

Reply:
How does it increases ambiguity? the name and java doc below specifies that it is the time in seconds after which credential will expire?

We need to return the expire time in seconds from the client library which is of the `GcpTokenResponse`, wanted to avoid repeated conversations, wdyt?

Reply:
This isn't a huge deal, but I think from the name it might be slightly unclear whether it's the number of seconds until it expires, or it's the timestamp at which it expires. If we want to avoid the conversion, can we can use a long and call it `expiresAfterSecs`, then convert it downstream instead? It seems like we're already doing two Long->String->Long conversions.

Reply:
ack, will send it as Instant.